lib/streamlit/elements/code.py

```diff
@@ -40,32 +40,57 @@ def code(
         Parameters
         ----------
         body : str
-            The string to display as code.
+            The string to display as code or monospace text.
```

Comment:
Just for clarity, these doc string changes probably come after a discussion with Debbie and are not directly related to the `st.text` PR, right?

Reply:
Yes! These were updated in #9592 (I may prefer to split the non-`st.text` changes out. But I also see how this change would affect other APIs as an entirety)

Reply:
yeah its fine to leave them in here 🙂 thanks for verifying!